GLAA.Services/LicenceApplication/LicenceApplicationViewModelBuilder.cs

```diff
@@ -36,6 +38,14 @@ public T New<T>() where T : new()
                 countyModel.Counties = referenceDataProvider.GetCounties();
             }
 
+            if (model is INeedStandards standardModel)
+            {
+                standardModel.Standards = statusRepository
```

Comment:
It's fine here for now but this could probably be moved to `IReferenceDataProvider`

Reply:
Yeah, I did think of that, but part of me wants to be able to get the data once per licence and apply it to all the models, rather than getting it once per model. Will note it for future enhancement